src/StreamFactory/GuzzleStreamFactory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Http Discovery package.
+ *
+ * (c) PHP HTTP Team <team@php-http.org>
+ *
+ * For the full copyright and license information, please read the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Http\Discovery\StreamFactory;
+
+use Http\Message\StreamFactory;
+use Psr\Http\Message\StreamInterface;
+
+/**
+ * Guzzle Stream Factory
```

Comment:
No, but I consider it to be noise as there is no added value in it. (It is exactly the same as the classname) I am going to remove it from other factories as well.